core/base/inc/DllImport.h

```diff
@@ -29,5 +29,12 @@
 # define R__EXTERN extern
 #endif
 
+#ifndef R__DLLEXPORT
+# ifdef _MSC_VER
+#  define R__DLLEXPORT __declspec(dllexport)
+# else
+#  define R__DLLEXPORT __attribute__ ((visibility ("default")))
```

Comment:
Is this really supported by all "our" compilers?

Reply:
It's already defined that way in several places in ROOT (`rootclingTCling.h`, `TCling.cxx`, ...), but I can also simply leave it empty...